mesa/batchrunner.py

```diff
@@ -22,8 +22,10 @@
 
 
 class ParameterError(TypeError):
-    MESSAGE = ('parameters must map a name to a value. '
-               'These names did not match paramerets: {}')
+    MESSAGE = (
+        "parameters must map a name to a value. "
+        "These names did not match paramerets: {}"
```

Comment:
Minor typo here

Reply:
#786 can't fix right away because the port for git is blocked in my current internet.